### PR TITLE
Add html and css files copied from previous Microverse admission project

### PR DESCRIPTION
**Pull request by Lasten-Ernest**
This pull request is for Adding html and css files copied from previous Microverse admission project and then review the project on my own.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    max-width: 500px;
+    margin: 0 auto 45px auto;
+    text-align: left;
+    line-height: 1.6;
+}
+
+li {
+    margin: 16px 0;
+    
+}
```
Comment (Lasten-Ernest) [inline index.css:34]: Please remove unnecessary line space. Your code does not look professional if you leave unnecessary line space.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+blockquote {
+    font-style: italic;
+    max-width: 545px;
+    margin: 0 auto 50px auto;
+    text-align: left;
+  }
+  
+
+img {
```
Comment (Lasten-Ernest) [inline index.css:44]: Please remember to use one line spacing for CSS code segment. It looks neat and professional.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            <li>
+              <strong>2008</strong> - Leaves his family to attend the
+              University of Malawi, where he graduated with a bachelor's degree in Computer science four years after.
+              
+            </li>
+            <li>
+              <strong>2013</strong> - He Joined a small company that creates software programs related 
+              to health where he worked as an intern in the IT department for one year and three months.  
+              
+            </li>
```
Comment (Lasten-Ernest) [inline index.html:35]: Please do not jump or leave line spaces anyhow unless when you want to indicate different sections of your program. It is a very good practice to only leave a line space or spaces when demarcating sections of your program.

**Review by Lasten-Ernest — COMMENTED** [review]
Please check my comments below and make required changes.

Request for re-review when you are done.
Thanks.

**Review by ishimwezachee — APPROVED** [review]
Hi @Lasten-Ernest , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**